app-misc/prog-express/prog-express-3.7.8.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils multilib
+
+DESCRIPTION="A modern and intuitive control software for the Batronix USB programming devices"
+HOMEPAGE="https://www.batronix.com"
+SRC_URI="amd64? ( http://www.batronix.com/exe/Batronix/Prog-Express/deb/${P}-1.amd64.deb )
+	x86? ( http://www.batronix.com/exe/Batronix/Prog-Express/deb/${P}-1.i386.deb )"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="prog-express"
+SLOT="0"
+
+RDEPEND="dev-libs/libusb
+	dev-db/sqlite
+	dev-dotnet/gtk-sharp
+	dev-dotnet/libgdiplus
+	dev-lang/mono
+	dev-lang/mono-basic
+	virtual/udev"
+DEPEND="app-arch/gzip
+	sys-devel/binutils"
+
+RESTRICT="strip"
+
+S="${WORKDIR}"
+
+QA_PREBUILT="usr/bin/bxusb usr/bin/bxusb-gui usr/bin/prog-express usr/sbin/bxfxload"
+
+src_unpack() {
```

Comment:
please do not handle this manually, use `unpacker.eclass` for this purpose

Reply:
Thanks, I will update the eBuild.